common/djangoapps/util/milestones_helpers.py

```diff
@@ -175,16 +174,20 @@ def get_required_content(course, user):
                 for content in milestone_path['content']:
                     required_content.append(content)
 
-    #local imports to avoid circular reference
+    # local imports to avoid circular reference
     from student.models import EntranceExamConfiguration
+    from courseware.access import has_access
     can_skip_entrance_exam = EntranceExamConfiguration.user_can_skip_entrance_exam(user, course.id)
-    # check if required_content has any entrance exam and user is allowed to skip it
+    # check if required_content has any entrance exam
+    # and user is allowed to skip it or user is member of staff
     # then remove it from required content
-    if required_content and getattr(course, 'entrance_exam_enabled', False) and can_skip_entrance_exam:
+    if required_content and getattr(course, 'entrance_exam_enabled', False) and \
+            (can_skip_entrance_exam or has_access(user, 'staff', course, course.id)):
```

Comment:
Seems like this `has_access` check should really have been incorporated into `user_can_skip_entrance_exam` - I _think_ that we are really saying that "*staff* users can skip entrance exam"; so it seems semantically out of place here